### ISSUE TITLE
Tracking issue for temporary test skips/xfails to revert

### ISSUE THREAD

**Issue by ev-br** [scipy/scipy#17278]
(As suggested in https://github.com/scipy/scipy/pull/17273#issuecomment-1288059660)

Once in a while, we skip or xfail tests to quickly restore the CI to the green status, when we either lack the immediate bandwidth to resolve an issue properly, or a fix takes a while etc. Once a proper fix lands, the skip needs to be reverted.
This is a tracking issue to keep track of these skips and their reverts. 

- [ ] gh-17273 skips a test of a discrete Riccatti equation which fails on Cirrus CI. The skip needs a revert once https://github.com/scipy/scipy/issues/16926 is resolved.

**Comment by andyfaff** [comment]
Thanks @ev-br should we pin it?

**Comment by andyfaff** [comment]
Unpin the pin in #17331

**Comment by lucascolley** [comment]
> https://github.com/scipy/scipy/pull/17331 Unpin a CI package dependency of pytest-xdist because of an incompatibility with pytest-cov

I think this point can be ticked off now, right? `pytest-xdist` is not pinned in the prerelease_deps job anymore.

**Comment by ev-br** [comment]
Ticked Riccati equation solvers in th OP, based on https://github.com/scipy/scipy/pull/20335 having removed the xfails. We might want to revisit the tolerance bumps at some point after openblas 3.26 becoming the lowest supporter version.

**Comment by lucascolley** [comment]
@ev-br what do you think about retiring, or at least unpinning this issue? I reckon our coverage of temporary skips/xfails is pretty poor, and we're probably fine tracking each instance in its own issue.

**Comment by ev-br** [comment]
Yeah, we can close this if it's not helpful anymore.

**Comment by lucascolley** [comment]
Should we open a separate issue for the unchecked box in the top post?

EDIT: gh-24000
